backend/src/main/java/ca/bc/gov/hlth/mohums/validator/BulkRemovalRequestValidator.java

```diff
@@ -0,0 +1,30 @@
+package ca.bc.gov.hlth.mohums.validator;
+
+import ca.bc.gov.hlth.mohums.model.BulkRemovalRequest;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Service
+public class BulkRemovalRequestValidator {
```

Comment:
I think it is good advice to instantiate it directly in the `UsersController`. I do not have plans to reuse it. The only tests I've written for this functionality are in `BulkRemovalIntegrationTests` I did not think it through and defaulted to @Service - probably if I were to write a unit test for this class, it would catch my attention - "Why do I need to load a whole Spring context for this simple class?"

I should apply the same principle to PermissionsValidator class (which checks just group management permission)